Change default such that we run on a stock Gentoo system.  Bug #438606

--- src/configure.default
+++ src/configure.default
@@ -674,8 +674,9 @@
 # BSD mailbox format. By default it will be run under the uid and gid of the
 # local user, and requires the sticky bit to be set on the /var/mail directory.
 # Some systems use the alternative approach of running mail deliveries under a
-# particular group instead of using the sticky bit. The commented options below
-# show how this can be done.
+# particular group instead of using the sticky bit. This is also the
+# default case on Gentoo, therefore group and mode are set below.
+# Comment them out, to get the default behaviour.
 
 local_delivery:
   driver = appendfile
@@ -683,8 +684,8 @@
   delivery_date_add
   envelope_to_add
   return_path_add
-# group = mail
-# mode = 0660
+  group = mail
+  mode = 0660
 
 
 # This transport is used for handling pipe deliveries generated by alias or
